src/Event/NotificationEvent.php

```diff
@@ -9,35 +9,152 @@
 
 namespace KevinPapst\TablerBundle\Event;
 
+use KevinPapst\TablerBundle\Model\NotificationV2Interface;
 use KevinPapst\TablerBundle\Model\NotificationInterface;
 
-class NotificationEvent extends ThemeEvent
+class NotificationEvent extends ThemeEvent implements NotificationEventInterface
```

Comment:
NotificationEvent cannot be replaced with custom implementation, so the interface can be removed

Reply:
```suggestion
class NotificationEvent extends ThemeEvent
```